packages/react/src/greyVest/Tag.js

```diff
@@ -27,9 +28,10 @@ let Tag = ({
       wordBreak: 'break-all',
       ...F.callOrReturn(tagStyle, value),
     }}
-    onClick={onClick}
+    onClick={onClick({ value, label })}
```

Comment:
This should be `onClick={() => onClick({ value, label })}`

Reply:
If it is a function call with no params in an event handler, should we not just use the named function though. That is what I learned a while back, and looking it up seems like it is still valid concern.

https://stackoverflow.com/questions/74434193/anonymous-function-vs-named-function-on-event-handlers

Reply:
I did make the change btw though